README.md

```diff
@@ -94,10 +94,13 @@ Since there is no way to guarantee that the sharee OCM provider will actually en
 
 ## Contributing
 
-The specification can be rendered as HTML documentation using [ReDoc](https://github.com/Redocly/redoc) and is available as follows:
+The specification has been rendered as HTML documentation using [ReDoc](https://github.com/Redocly/redoc), and is available as follows:
```

Comment:
It gets rendered on-the-fly from spec.yml when you access those URLs, so I think 'can be' is more accurate than 'has been'?

Reply:
That's a fair point! (I thought about the previous tagged versions but even those ones are rendered on the fly indeed...)